src/routes/userRoutes.js

```diff
@@ -7,6 +7,7 @@ let app = express();
 let config = require('../config');
 let router = express.Router();
 
+// safer to get the secret from an env variable - Harold
```

Comment:
You can create your own environment variable. There is a way to set up environment variables for heroku. It might be something to share with the whole class once one team tackles this. Let me know when you want to try.